mirrormanager2/xmlrpc.py

```diff
@@ -41,7 +42,11 @@
 
 @XMLRPC.register
 def checkin(pickledata):
-    config = pickle.loads(bz2.decompress(base64.urlsafe_b64decode(pickledata)))
+    uncompressed = bz2.decompress(base64.urlsafe_b64decode(pickledata))
+    try:
+        config = json.loads(uncompressed)
+    except ValueError:
+        config = pickle.loads(uncompressed)
```

Comment:
Should we file a reminder issue somewhere so we don't forget to remove the pickle support soon?